src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/services/holidaydates/HolidayServiceTest.java

```diff
@@ -17,20 +17,20 @@ class HolidayServiceTest {
 
     private HolidayService holidayService;
 
-    private static List<LocalDate> HOLIDAYS = List.of(
+    private static final List<LocalDate> listHOLIDAYS = List.of(
         LocalDate.of(2022, Month.DECEMBER, 25),
         LocalDate.of(2022, Month.JANUARY, 1),
         LocalDate.of(2022, Month.APRIL, 15)
     );
 
     @BeforeEach
     void setup() {
-        holidayService = new HolidayService(HOLIDAYS);
+        holidayService = new HolidayService(listHOLIDAYS);
```

Comment:
Is it a best idea to change this variable name to camel case?

Reply:
The hmcts sonar config specifies that it wants camelCase in the lower+upper format. I could change it to listHolidays to be more in line with that. This is the regex that gets used

Rename this field "HOLIDAYS" to match the regular expression '^[a-z][a-zA-Z0-9]*$'.

Reply:
Can we put 'holidaysList' ? 
@raghera Please correct me if I am wrong.

Reply:
I think in most cases we've used the naming convention for collections to either be pluralised or appending `List`.  So either `holidays` or `holidayList` is fine in this case.

Caps are usually for constant values.